tests/New_ATS/Tests/routeguide/testcases/RecordRoute.json

```diff
@@ -0,0 +1,64 @@
+[
+  {
+    "input": [
+      {
+        "latitude": 1,
+        "longitude": 2
+      },
+      {
+        "latitude": 3,
+        "longitude": 4
+      },
+      {
+        "latitude": 5,
+        "longitude": 6
+      }
+    ],
+    "output": {
+      "point_count": 3,
+      "feature_count": 3,
+      "distance": 0,
```

Comment:
once you modify RecordRoute event case to return values, match them with the expected output here